Redesign send search

musidlo opened the pull request.

musidlo commented on packages/app/components/SearchBar.tsx:
```diff
@@ -477,16 +506,32 @@ function Search({ label, placeholder = 'Sendtag, Phone, Send ID, Address' }: Sea
+                iconAfter: query && (
+                  <Stack onPress={handleClearClick} cursor={'pointer'}>
+                    <IconX color={'$silverChalice'} size="$1" />
+                  </Stack>
```
alright, you are the boss here, changed it to be a button

youngkidwarrior commented:
![image](https://github.com/user-attachments/assets/84e8b9b3-d51d-44da-9816-00b75b386c59)

Light mode is a little hard to read. Let's darken the placeholder text

musidlo commented:
Hey @youngkidwarrior, darkened placeholder color on light mode, also as Ethen asked, I removed italics from placeholder, bumped font size of it and modified search icon

youngkidwarrior commented on packages/app/features/send/components/SendTopNav.tsx:
```diff
@@ -57,7 +57,7 @@ export function SendTopNav() {
                   return 'Enter Amount'
                 default:
-                  return 'Select Recipient'
+                  return 'Send'
```
One more thing.

 hide the Send header on desktop. Align the search bar with top of other screens
